src/middleware.ts

```diff
@@ -12,6 +12,13 @@ export function middleware(request: NextRequest) {
 
   const requestHeaders = new Headers(request.headers);
   requestHeaders.set("x-nonce", nonce);
+
+  if (process.env.E2E_TEST_ENV === "local") {
+    const forcedFeatureFlags =
+      request.nextUrl.searchParams.get("feature_flags");
+    requestHeaders.set("x-forced-feature-flags", forcedFeatureFlags ?? "");
+  }
+
```

Comment:
Instead of adding runtime middleware code, we could also skip that and just directly call [`.setExtraHTTPHeaders`](https://playwright.dev/docs/api/class-page#page-set-extra-http-headers), I think?

Reply:
Oh yeah — a lot better: [7f3feea](https://github.com/mozilla/blurts-server/pull/5150/commits/7f3feeaca6ffce562b3ab32e7319ebe97b26fda4).